src/server.rs

```diff
@@ -369,11 +390,14 @@ async fn list(config: web::Data<RwLock<Config>>) -> Result<HttpResponse, Error>
                 } else {
                     None
                 };
-                Some(ListItem {
-                    file_name,
-                    file_size: metadata.len(),
-                    expires_at_utc,
-                })
+                Some(
+                    ListItem {
+                        file_name,
+                        file_size: metadata.len(),
+                        creation_date,
```

Comment:
the uts2ts crate is not TZ aware, thus all timestamps are UTC.
TZ calculations are complex and we decided back then not to use the `time` or `chrono` crate because they pull in a lot of dependencies.

thus this should be named consistently with the `expires_at_utc`:

```suggestion
                        creation_date_utc,
```